examples/Bunch/include/simulation_defines/param/gasConfig.param

```diff
@@ -46,35 +52,84 @@ namespace SI
-     * which is done in the first cell of the simulation
-     * unit: cells
-     */
-    (PMACC_C_VALUE(uint32_t, vacuumCellsY, 50))
+        /** height of vacuum area on top border
+         *
+         * this vacuum is important because of the laser initialization,
+         * which is done in the first cell of the simulation
+         * unit: cells
+         */
+        (PMACC_C_VALUE(uint32_t, vacuumCellsY, 50))
 
-    /** The central position of the gas distribution
-     *  unit: meter
-     */
-    (PMACC_C_VECTOR_DIM(float_64, simDim, center_SI, 1.024e-5, 9.072e-5, 1.024e-5))
+        /** The central position of the gas distribution
+         *  unit: meter
+         */
+        (PMACC_C_VECTOR_DIM(float_64, simDim, center_SI, 1.024e-5, 9.072e-5, 1.024e-5))
+
+        /** the distance from gasCenter_SI until the gas density decreases to its 1/e-th part
+          *  unit: meter */
+        (PMACC_C_VECTOR_DIM(float_64, simDim, sigma_SI, 6.0e-6, 6.0e-6, 6.0e-6))
+    ); /* struct GaussianCloudParam */
+
+    /* definition of gas cloud profile */
+    typedef GaussianCloudImpl< GaussianCloudParam > GaussianCloud;
+
+
+    struct FreeFormulaFunctor
+    {
+        /** This formula uses SI quantities only
+         *
+         * The profile will be multiplied by GAS_DENSITY.
+         *
+         * @param position_SI total offset including all slides [in meter]
+         * @param cellSize_SI cell sizes [in meter]
+         *
+         * @return float_X density [normalized to 1.0]
+         */
+        HDINLINE float_X operator()(
+            const floatD_64& position_SI,
+            const float3_64& cellSize_SI
+        )
+        {
+            /* add particle in cell at center of Gaussian Cloud profile */
+            const float3_64 position_start_SI( 1.024e-5, 9.072e-5, 1.024e-5 );
+
+            /* from all cells ... */
+            const PMacc::math::UInt64< simDim > cell_id( position_SI / cellSize_SI.shrink< simDim >() );
+
+            /* ... we calculate the corresponding "center" cell to init the particle in ... */
+            const PMacc::math::UInt64< simDim > cell_start(
+                precisionCast< uint64_t >(
+                    math::floor(
+                        position_start_SI.shrink< simDim >() /
+                        cellSize_SI
+                    )
+                )
+            );
+
+            /* ... and only in that center cell the density is 1.0, outside zero */
+            bool isStartCell = true;
+            for( uint64_t d = 0; d < simDim; ++d )
+                if( cell_id[d] != cell_start[d] )
+                    isStartCell = false;
+
+            if( isStartCell )
+                return 1.0;
 
-    /** the distance from gasCenter_SI until the gas density decreases to its 1/e-th part
-      *  unit: meter */
-    (PMACC_C_VECTOR_DIM(float_64, simDim, sigma_SI, 6.0e-6, 6.0e-6, 6.0e-6))
-); /* struct GaussianCloudParam */
+            return 0.0;
```

Comment:
imho, that is a question of personal preference.
since they is no code obfuscating what is happening, I will keep it that way if we do not find something more to change along.